treeherder/push_health/linting.py

```diff
@@ -17,3 +17,12 @@ def get_lint_failures(push, parent_push=None):
         mark_failed_in_parent(failures, get_lint_failures(parent_push)[1])
 
     return (result, failures)
+
+
+def get_lint_in_progress_count(push):
+    return Job.objects.filter(
+        Q(machine_platform__platform='lint') | Q(job_type__symbol='mozlint'),
+        push=push,
+        tier__lte=2,
+        result='unknown',
+    ).count()
```

Comment:
Do we need a specific unknown count? :) I don't think we have this metric anywhere else in Push Health. If we do need it, maybe we could reuse some of the same logic in `get_lint_failures` so we don't have duplicate queries for each of the tests, builds and lints which I'd think would make this API even slower?

Reply:
The `result` of `unknown` is what we have when the task is not yet completed.  So it's a simpler way of checking if things are in-progress than saying `result__in=['pending', 'running']`.  I suppose I could have just tested for `.exclude(state='completed')`.  And perhaps that's less confusing.  Happy to switch if you think that's better.  :)

Reply:
Ah, what I meant was we're already checking for 'unknown' in `get_job_results`, which is called on line 14 of this file. So maybe we can reuse some of that logic instead of doing another query?

Reply:
So, it returns result and failures, the latter we only look at if the result is "fail" (versus "pass" or "unknown").

Reply:
I have to call it a day, but I'll take a look tomorrow.

Reply:
I see what you mean.  Yeah, I modified this for the `linting` and `builds` sections, as you suggested.  For tests, it's a little trickier, because we're querying based on FailureLInes, and not `result` so I opted to keep it a separate query.  How does that sound to you?

Reply:
Does it affect query time at all? Since only the result of 'unknown' and not the count of 'unknowns' is needed for MyPushes and and Health views maybe the unknown count could be optional based on a passed query param?

Reply:
I don't think the cost is great, but "a penny saved..." and so forth.  :). So I'll add a param to skip the call in the `MyPushes` page.  I followed the convention of the `with_history`.  `with_in_progress_tests` is a bit long-winded, but should be easy to understand, I hope.

Reply:
sounds good :)